tiledb/common/random/random_label.h

```diff
@@ -33,10 +33,90 @@
 #ifndef TILEDB_HELPERS_H
 #define TILEDB_HELPERS_H
 
+#include <iomanip>
+#include <mutex>
+#include <sstream>
 #include <string>
 
+#include "tiledb/common/exception/exception.h"
+#include "tiledb/common/random/prng.h"
+#include "tiledb/sm/misc/tdb_time.h"
+
 namespace tiledb::common {
 
+class RandomLabelException : public StatusException {
+ public:
+  explicit RandomLabelException(const std::string& message)
+      : StatusException("RandomLabel", message) {
+  }
+};
+
+class RandomLabelGenerator {
```

Comment:
Add doc string with expectations, especially that it has 4 bytes of ordered sequence from a random-initialized starting point.